apps/system/js/audio_volume/linux.js

```diff
@@ -47,7 +47,7 @@ async function setupAudio() {
   }
 
   // Initialize the HW to the current value.
-  await setAudioVolume(currentVolume);
+  // await setAudioVolume(currentVolume);
```

Comment:
unrelated change, please revert in this PR

Reply:
alright